Controller/Admin/TransactionTab.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Wirecard\Oxid\Controller\Admin;
+
+use Wirecard\Oxid\Model\Transaction;
+
+use OxidEsales\Eshop\Core\Registry;
+use OxidEsales\Eshop\Application\Controller\Admin\AdminDetailsController;
+use OxidEsales\Eshop\Application\Model\Order;
+use OxidEsales\Eshop\Application\Model\Payment;
+
+/**
+ * Controls the view for a single transaction tab.
+ */
+class TransactionTab extends AdminDetailsController
+{
+    protected $_sThisTemplate = 'transaction_tab.tpl';
+
+    protected $_i18n = null;
+
+    /**
+     * TransactionTab constructor.
+     */
+    public function __construct()
+    {
+        parent::__construct();
+        
+        $this->_i18n = Registry::getLang();
+    }
+
+    /**
+     * @inheritdoc
+     *
+     * @return string
+     */
+    public function render(): string
+    {
+        $objectId = $this->getEditObjectId();
+
+        if (isset($objectId) && $objectId !== '-1') {
+            $transaction = oxNew(Transaction::class);
```

Comment:
In my opinion, having a separate private `_getListData` function would make this one more readable by simplifying it.